File: utils/whatsapp_numero.py
import re


def whatsapp_numero(numero):
    """Limpia el numero de WhatsApp guardado en admin (puede tener espacios, guiones,
    parentesis o venir sin el indicativo de pais) para que el link a wa.me funcione."""
    if not numero:
        return None
    solo_digitos = re.sub(r'\D', '', numero)
    if len(solo_digitos) == 10:
        return '57' + solo_digitos  # celular colombiano sin indicativo
    return solo_digitos

File: src/utils/whatsappNumero.js
// Limpia el numero de WhatsApp guardado en admin (puede tener espacios, guiones,
// parentesis o venir sin el indicativo de pais) para que el link a wa.me funcione.
module.exports = function whatsappNumero(numero) {
    if (!numero) return null;
    const soloDigitos = numero.replace(/\D/g, '');
    if (soloDigitos.length === 10) return '57' + soloDigitos; // celular colombiano sin indicativo
    return soloDigitos;
};
